core/src/main/csharp/ch/cyberduck/core/diagnostics/TcpReachability.cs

```diff
@@ -39,12 +39,16 @@ static TcpReachability()
 
         public bool isReachable(Host h)
         {
-            switch ((Scheme.__Enum)h.getProtocol().getScheme().ordinal())
+            var scheme = h.getProtocol().getScheme();
+            if (Scheme.file.equals(scheme))
```

Comment:
This doesn't need to be in a try-block, reducing complexity later.

Reply:
Should we fast-exit for Hostname equals `""`?

Reply:
On macOS, no reachability check is made if the input is blank [^1]. 

[^1]: https://github.com/iterate-ch/cyberduck/blob/da4384c42831e87569bd3bc9ea1835c3399612d0/osx/src/main/java/ch/cyberduck/ui/cocoa/controller/BookmarkController.java#L287

Reply:
Ah I see. Applying to Mountain Duck as well.